.github/workflows/test-internet.yml

```
@@ -43,7 +43,7 @@ permissions:
 
 jobs:
   test-internet:
-    if: github.repository == 'nodejs/node' || github.event_name != 'schedule'
+    if: (github.repository == 'nodejs/node' || github.event_name != 'schedule') && github.event.pull_request.draft == false
```

To me, it would be easier to follow if written as a ternary (If it's a schedule run, check if we're on nodejs/node, otherwise check if it's a draft PR)
```suggestion
    if: github.event_name == 'schedule' && github.repository == 'nodejs/node' || github.event.pull_request.draft == false
```